packages/react/roving-focus/src/RovingFocusGroup.tsx

```diff
@@ -198,6 +211,8 @@ const RovingFocusItem = React.forwardRef((props, forwardedRef) => {
           // We prevent focusing non-focusable items on `mousedown`.
           // Even though the item has tabIndex={-1}, that only means take it out of the tab order.
           if (!focusable) event.preventDefault();
+          // Safari doesn't focus a button when clicked so we run our logic on mousedown also
```

Comment:
It occurred to me whilst writing this comment (and the other one too) that we do this because of buttons and mention buttons, even though this render a `span` by default. So there's a bit of a disconnect here. It's obviously the neatest place to fix it, but how can we bridge the gap a bit better in terms of explanation?

Reply:
Oh yeah... that's a bit poop but can't think of a better way of handling it so 🤷